core/modules/layout/plugins/access/user_compare_layout_access.inc

```diff
@@ -0,0 +1,97 @@
+<?php
+/**
+ * @file
+ * Contains UserCompareLayoutAccess class.
+ */
+
+/**
+ * Plugin to provide access if two user account contexts are the same.
+ */
+class UserCompareLayoutAccess extends LayoutAccess {
+  /**
+   * Constructor for a Layout access rule.
+   */
+  public function __construct($plugin_name, array $data = array()) {
+    parent::__construct($plugin_name, $data);
+    $this->settings += array(
+      'equality' => 1,
+    );
+  }
+
+  /**
+   * {@inheritdoc}
+   */
+  public function summary() {
+    $contexts = $this->settings['contexts'];
+    if (empty($contexts) || count($contexts) != 2 || empty($contexts['user1']) || empty($contexts['user2'])) {
+      return t('Compares two user accounts');
+    }
+
+    $labels = array();
+    foreach ($this->settings['contexts'] as $key => $source) {
+      if ($source == 'current_user') {
+        $labels[$key] = t('The logged-in user account');
+      }
+      elseif (is_numeric($source)) {
+        $message = 'The user account from the path (in position @position)';
+        $substitutions = array('@position' => $position = $source + 1);
+        $labels[$key] = t($message, $substitutions);
+      }
+      else {
+        $labels[$key] = t('An unknown user account');
+      }
+    }
+
+    $comparison = !empty($this->settings['equality']) ? 'is the same as' : 'is different from';
+    return t('@id2 @comp the Logged-in user account.', array(
```

Comment:
This needs to be separated out as two separate `t()` calls. With this approach `is the same as` and `is different from` is not translatable, and should be included as part of the `t()` string.